HelperMethods.rb

```diff
@@ -28,9 +31,11 @@ def change_password
     @wait.until {@driver.find_element(:id, 'my_account_form').displayed? }
     @driver.find_element(:class, 'icon-passwd').click
     @wait.until {@driver.find_element(:css, '.controller-my.action-password').displayed? }
-    @driver.find_element(:id, 'password').send_keys 'Password1'
-    @driver.find_element(:id, 'new_password').send_keys 'NewPassword1'
-    @driver.find_element(:id, 'new_password_confirmation').send_keys 'NewPassword1'
+    @driver.find_element(:id, 'password').send_keys @password
+    user = Users.new
+    new_password = user.password
```

Comment:
this line is redundant line
you could just point to user.password in the next one